### PR TITLE
Add support for listing leaderboards

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

File: Runtime/Game/Requests/EntitlementRequests.cs.meta [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+guid: 930b5026c070b2d4fb183ec255103c0d
+MonoImporter:
+  externalObjects: {}
+  serializedVersion: 2
+  defaultReferences: []
+  executionOrder: 0
+  icon: {instanceID: 0}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant:
```
Comment (AlmightyMikkel) [inline Runtime/Game/Requests/EntitlementRequests.cs.meta:11]: Did you change anything in the Entitlement file?

File: Runtime/Game/Requests/LeaderboardRequest.cs [hunk 2]
```diff
@@ -7,6 +7,50 @@ namespace LootLocker.Requests
...
+        /// <summary>
+        /// Schedule of the Leaderboard.
+        /// </summary>
+        public LootLockerLeaderboardSchedule schedule { get; set; }
+        /// <summary>
+        /// A List of rewards tied to the Leaderboard.
+        /// </summary>
+        public LootLockerLeaderboardReward[] rewards { get; set; }
+    }
+
```
Comment (AlmightyMikkel) [inline Runtime/Game/Requests/LeaderboardRequest.cs:53]: Was there any changes here? I can't figure out why its highlighted

Comment (kirre-bylund) [reply Runtime/Game/Requests/LeaderboardRequest.cs:53]: You'd written it as a LootLockerResponse in LootLockerLeaderboardDetails.cs, I copied it here as a pure data struct without changes.

**Comment by kirre-bylund** [comment]
Needs to be tested before merging

File: Runtime/Game/Requests/LeaderboardRequest.cs (hunk 2 above)
Comment (AlmightyMikkel) [reply Runtime/Game/Requests/LeaderboardRequest.cs:53]: Aah gotcha ty!

File: Runtime/Game/Requests/EntitlementRequests.cs.meta (hunk 1 above)
Comment (kirre-bylund) [reply Runtime/Game/Requests/EntitlementRequests.cs.meta]: Fixed

**Comment by kirre-bylund** [comment]
Tested and working

**Review by AlmightyMikkel — APPROVED** [review]
LGTM

**Pull request merged**